x/ibc-transfer/spec/README.md

```diff
@@ -0,0 +1,22 @@
+<!--
+order: 0
+title: IBC Overview
+parent:
+  title: "ibc-transfer"
+-->
+
+# `ibc-transfer`
+
+## Abstract
+
+This paper defines the implementation of the ICS20 protocol on the Cosmos SDK.
+
+For the general specification please refer to the [ICS20 Specification](https://github.com/cosmos/ics/tree/master/spec/ics-020-fungible-token-transfer).
+
+## Contents
+
+1. **[Concepts](01_concepts.md)**
+2. **[State](02_state.md)**
+3. **[Messages](04_messages.md)**
+4. **[Events](06_events.md)**
```

Comment:
why the gaps? can we create the files?

Reply:
I wasn't really sure what to do here because there is a gap in the IBC spec. I don't think callbacks are necessary, right? Do the numbers get bumped for files not needed?

Reply:
updated